routes/api.js

```diff
@@ -1,10 +1,28 @@
 const express = require('express');
 const router = express.Router();
 
+const models = require('../database/models');
+
+const localUser = models.user_account;
 /* GET api listing. */
 router.get('/getUserInfo', (req, res) => {
     res.status(200).json({user: req.user});
 });
 
-module.exports = router;
+router.get('/username', (req, res) => {
+    const email=req.user.email;
```

Comment:
It is not different, the PR #21 had that because pcpolicycomonent uses navbar which uses this file.  Also I cherry picked in order to submit 3 PR so all three PRs have same base branch.